target/bin/listmailuser

```diff
@@ -8,6 +8,10 @@ source /usr/local/bin/helpers/index.sh
 source /etc/dms-settings 2>/dev/null
 
 function _main() {
+  if [[ ${ACCOUNT_PROVISIONER} != 'FILE' ]]; then
+    _exit_with_error "This command is only compatible with 'ACCOUNT_PROVISIONER=FILE'"
+  fi
```

Comment:
This could technically be dropped, but I don't think DMS is intended to support both file based accounts and LDAP at the same time, so I'm inclined to keep the early exit constraint here? 🤔

Reply:
Fail-fast is the right approach here, indeed.